libpod/lock/shm/shm_lock_test.go

```diff
@@ -28,8 +30,11 @@ const lockPath = "/libpod_test"
 
 // We need a test main to ensure that the SHM is created before the tests run
 func TestMain(m *testing.M) {
-	// Remove prior /dev/shm/libpod_test
-	os.RemoveAll("/dev/shm" + lockPath)
+	// Remove prior /libpod_test
+	if err := unlinkSHMLock(lockPath); err != nil && !errors.Is(err, fs.ErrNotExist) {
+		fmt.Fprintf(os.Stderr, "Error cleaning SHM for tests: %v\n", err)
+		os.Exit(-1)
```

Comment:
I don't think unit test should use os.Exit(), t.Fatal() should be enough here.

Reply:
This was following the existing pattern in this function. Also, there isn't a t variable since this is the test main. I changed the second call site for unlinkSHMLock further down to use t.Fatalf since this one is in a test.